streamflow/cwl/processor.py

```diff
@@ -232,6 +233,7 @@ async def _process_file_token(
                 token_value = await utils.update_file_token(
                     context=self.workflow.context,
                     connector=connector,
+                    cwl_version=cast(CWLWorkflow, self.workflow).cwl_version,
```

Comment:
This should not be cast here: all cwl-related functions should accept a `CWLWorkflow` object instead of a simple `Workflow` for coherence.